lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -504,9 +504,11 @@ void SwiftLanguageRuntimeImpl::ProcessModulesToAdd() {
   // Add all defered modules to reflection context that were added to
   // the target since this SwiftLanguageRuntime was created.
   modules_to_add_snapshot.ForEach([&](const ModuleSP &module_sp) -> bool {
-    AddModuleToReflectionContext(module_sp);
-    progress.Increment(++completion,
-                       module_sp->GetFileSpec().GetFilename().AsCString());
+    if (module_sp) {
+      AddModuleToReflectionContext(module_sp);
+      progress.Increment(++completion,
+                         (std::string)module_sp->GetFileSpec().GetFilename());
```

Comment:
I strongly suggest avoiding C-style casts, few people can actually state the rules the compiler follow for those in C++...  we can just do: `module_sp->GetFileSpec().GetFilename().GetString()`